### PR TITLE
Core login integration

### PR DESCRIPTION
**Pull request by terlan98**
Reusing login-related functionality from ArtemisCore. This is needed to later share the service layer between apps.

Please merge #156 before this.

### CODE REVIEW

**Comment by terlan98** [comment]
Changed base branch to `chore/refactor` to make reviewing easier. Please change back to `develop` while merging

File: Themis.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -123,6 +136,10 @@
 /* Begin PBXFileReference section */
 		0BEC4C34292EA1A400F9F802 /* FiletreeSidebarView.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = FiletreeSidebarView.swift; sourceTree = "<group>"; };
 		0BEC4C36292EA32A00F9F802 /* CodeEditorView.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = CodeEditorView.swift; sourceTree = "<group>"; };
+		65153F9C2A0F97F70020B6FE /* artemis-ios-core-modules */ = {isa = PBXFileReference; lastKnownFileType = wrapper; name = "artemis-ios-core-modules"; path = "../artemis-ios-core-modules"; sourceTree = "<group>"; };
```
Comment (maximiliansoelch) [inline Themis.xcodeproj/project.pbxproj]: This shouldn't be here, right?

**Review by maximiliansoelch — CHANGES_REQUESTED** [review]
Code looks good 👍
However, locally the build fails for me, so I can't test + merge conflicts with develop.

**Review by maximiliansoelch — APPROVED** [review]
Now tested locally, login still works as expected after the changes 👍

**Pull request merged**